### PR TITLE
Update secret.md

### PR DESCRIPTION
**Pull request by lvtaochina**
a minor Chinese translation modification for the term "#create-a-pod-that-has-access-to-the-secret-data-through-a-volume"
old:使用 Secret 安全地分发凭据
new:创建一个通过volume来访问Secret的Pod
The reason is that the old one is not a correct Chinese translation

### CODE REVIEW

File: content/zh-cn/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -989,7 +989,7 @@ For instructions, refer to
 [Distribute credentials securely using Secrets](/docs/tasks/inject-data-application/distribute-credentials-secure/#create-a-pod-that-has-access-to-the-secret-data-through-a-volume).
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/configuration/secret.md:989]: #47250 is merged. Line 989 shall be updated too.

**Comment by T-Lakshmi** [comment]
@lvtaochina,
Can you please rebase the PR to include latest updates.

**Review by Arhell — COMMENTED** [review]
lgtm

**Comment by mengjiao-liu** [comment]
/hold

The latest push may not have any changes(only an empty line). Is that right?

**Comment by windsonsea** [comment]
Only a blank line added currently. This PR will be closed. Feel free to re-open it if you have some more content to add, thanks.

/close

**Pull request closed without merging**